tensorboard/webapp/widgets/line_chart_v2/lib/scale.ts

```diff
@@ -74,7 +74,9 @@ class LinearScale implements Scale {
     const padding =
       max === min
         ? // In case both `min` and `max` are 0, we want some padding.
-          Math.max(min * PADDING_RATIO, MIN_SIGNIFICANT_PADDING)
+          min === 0
+          ? PADDING_FOR_ZERO
+          : Math.abs(min) * PADDING_RATIO
```

Comment:
FWIW, I'm ok with this padding approach, but historically we had added more generous padding for constant plots (basically using `[0, 2c]` as the range) to ensure that we were always showing the x axis, while still adjusting the overall scale to match the magnitude of the constant.  Did you have a preference for the tighter zoom?  Could see an argument either way, just curious.

Reply:
I see some reference to that time, but I am not sure whether that's what we have today (comment does not agree with the code, at least not in an obvious way).

https://github.com/tensorflow/tensorboard/blob/c7a4c73ca389318b200de157dda253f07279eeb8/tensorboard/components/vz_line_chart2/linear-scale.ts#L45-L50 

Regardless of what is happening in vz_line_chart, decided to include `0` in all cases.